### ISSUE TITLE
bug: “active” class does not work in <ion-item href=“”>

### ISSUE THREAD

**Issue by sgon00** [driftyco/ionic#3050]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>all</span>  

<span ionic-description>Please check the code:
http://codepen.io/anon/pen/PwOQwR

```html
<ion-item class="item-dark active" nav-clear ng-click="login()">
 Login
</ion-item>
<ion-item class="item-dark active" nav-clear href="#">
Browse
</ion-item>
```

`active` class works when the `<ion-item>` does not have `href` attribute.
When `<ion-item>` has the `href` attribute, it will add `<a>` inside ion-item and `<a>`'s background color will override `active` background color with:

```css
.item-complex.item-dark > .item-content {
  border-color: #111;
  background-color: #444;
  color: #fff;
}
```

To workaround this issue, I have to add the following css style to override `<a>` background color again:

```css
.item-complex.item-dark.active > .item-content {
    border-color: #000;
    background-color: #262626;
}
```

**Comment by Ionitron** [comment]
Greetings @sgon00!

My sensors indicate that you need to update your issue through our [custom issue form](http://ionicframework.com/submit-issue/#/?iid=3050). We are now requiring all issues to be submitted this way, to ensure that we have all of the information necessary to fix them as quickly as possible.

**[Click Here To Update Your Issue](http://ionicframework.com/submit-issue/#/?iid=3050)**

I will have no choice but to close this issue if it is not resubmitted through the form. Please fill out the rest of the form, so that I may use my friendly robot powers to assist you.

Thank you!

<span ionitron-issue-resubmit></span>

**Comment by djett41** [comment]
I believe this is related to the same issue I was seeing.  In beta 14, line 3100 of ionic.js, the ionic.activator.start is supposed to be checking for elements with .item class when it walks up parents, but eleToActive will ALWAYS be undefined in this condition so the check will never pass...

Therefore to get the activated class added to your item, your ion-item will have to have an ng-click which would pass the third condition.

**Comment by mhartington** [comment]
@sgon00 it does work, it may just be a bug with the colored items (positive, calm, dark)

http://codepen.io/mhartington/pen/bNMXbZ

Notice that a normal item accepts the active class without any issues. 
Will look into this more.

**Comment by perrygovier** [comment]
Fixed in 9c378633e257ac77cbb2dc7723bfb22e35bf39bc
